packages/node/package.json

```diff
@@ -31,6 +31,9 @@
     "app-module-path": "^2.2.0",
     "dayjs": "^1.10.5",
     "lodash": "^4.17.21",
+    "merkle-mountain-range": "^2.0.0",
+    "merkle-tools": "^1.4.1",
+    "object-hash": "^2.2.0",
```

Comment:
is this the lib you think can help with generate hash for object? I can't find it been used anywhere.